pkg/scale/decode.go

```diff
@@ -344,7 +344,17 @@ func (ds *decodeState) decodeVaryingDataTypeSlice(dstv reflect.Value) (err error
 
 func (ds *decodeState) decodeCustomVaryingDataType(dstv reflect.Value) (err error) {
 	initialType := dstv.Type()
-	converted := dstv.Convert(reflect.TypeOf(VaryingDataType{}))
+	var converted reflect.Value
```

Comment:
these changes should be in a different PR if this isn't working.

Reply:
I just checked, you need to rebase.  These changes were added to scale a while ago.

Reply:
oh beautiful!